### PR TITLE
Typo fixes in repository documentation

### PR DESCRIPTION
**Pull request by ItsAdityaKSingh**
*Added a few commas and removed them at some places. Fixed a few typos and sentence fragments. Minute punctuation changes*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Thanks for the submission, but we can’t proceed with reviewing this PR until the CLA is signed. Please let us know once you’ve signed the CLA.

**Comment by ItsAdityaKSingh** [comment]
@stuartmorgan, I have signed the CLA! You may now proceed with merging :)

**Comment by stuartmorgan** [comment]
> You may now proceed with merging :)

As explained in the contributing docs that you've indicated via the PR checklist that you have read, the next step is a review, not merging.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Additional resources specific to the plugins repository:
   which covers the setup process for this repository.
 - [Plugins repository structure](https://github.com/flutter/flutter/wiki/Plugins-and-Packages-repository-structure),
   to get an overview of how this repository is laid out.
-- [Plugin tests](https://github.com/flutter/flutter/wiki/Plugin-Tests), which explains
+- [Plugin tests](https://github.com/flutter/flutter/wiki/Plugin-Tests), which explain
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:18]: This change is incorrect. The page is singular.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ Additional resources specific to the plugins repository:
...
   the different kinds of tests used for plugins, where to find them, and how to run them.
   As explained in the Flutter guide,
-  [**PRs needs tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
+  [**PRs need tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
   this is critical to read before submitting a PR.
 - [Contributing to Plugins and Packages](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages),
-  for more information about how to make PRs for this repository, especially when
+  for more information about making PRs for this repository, especially when
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:24]: Please don't make arbitrary stylistic changes.

File: README.md [hunk 3]
```diff
@@ -8,8 +8,7 @@ repo](https://github.com/flutter/flutter). It contains the source code for
 Flutter first-party plugins (i.e., plugins developed by the core Flutter team).
 Check the `packages` directory for all plugins.
 
-Flutter plugins enable access to platform-specific APIs. For more information
-about plugins, and how to use them, see
+Flutter plugins enable access to platform-specific APIs. For more information about plugins and how to use them, see
```
Comment (stuartmorgan) [inline README.md:11]: This is a purely subjective stylistic change.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ItsAdityaKSingh) [reply README.md:11]: I have removed an unnecessary comma here. If you wish, I could revert it.

**Comment by ItsAdityaKSingh** [comment]
Thanks a lot for your review, @stuartmorgan. I have reversed the changes mentioned by you.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,10 +18,10 @@ Additional resources specific to the plugins repository:
 - [Plugin tests](https://github.com/flutter/flutter/wiki/Plugin-Tests), which explains
   the different kinds of tests used for plugins, where to find them, and how to run them.
   As explained in the Flutter guide,
-  [**PRs needs tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
+  [**PRs need tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
   this is critical to read before submitting a PR.
 - [Contributing to Plugins and Packages](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages),
-  for more information about how to make PRs for this repository, especially when
+  for more information about 'how to make' PRs for this repository, especially when
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:24]: I don't understand this change; it makes no grammatical sense to me.

File: README.md (hunk 3 above)
Comment (stuartmorgan) [reply README.md:11]: I understand the change you made, I'm saying it's a purely subjective stylistic change. There was nothing wrong with it before, it just changes the emphasis very slightly. Unlike the typo fix in the other file, this isn't fixing a problem, it's just making it match how you would choose to write it.

Please don't make purely stylistic changes to docs. If everyone did that, we'd be constantly churning docs for no reason.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

**Comment by ItsAdityaKSingh** [comment]
Sure. I understand your complaint. So should I reverse the removed comma here?

File: CONTRIBUTING.md (hunk 4 above)
Comment (ItsAdityaKSingh) [reply CONTRIBUTING.md:24]: Removing it back :)

**Comment by stuartmorgan** [comment]
Yes.

**Comment by ItsAdityaKSingh** [comment]
Done @stuartmorgan :)

File: README.md [hunk 5]
```diff
@@ -8,8 +8,7 @@ repo](https://github.com/flutter/flutter). It contains the source code for
 Flutter first-party plugins (i.e., plugins developed by the core Flutter team).
 Check the `packages` directory for all plugins.
 
-Flutter plugins enable access to platform-specific APIs. For more information
-about plugins, and how to use them, see
+Flutter plugins enable access to platform-specific APIs. For more information about plugins, and how to use them, see
```
Comment (stuartmorgan) [inline README.md:11]: You're now removing line wrapping for no reason.

We appreciate the typo fix, as that is a legitimate issue in the documentation, but having lots of review cycles for unrelated irrelevant changes isn't a good use of time. If you can restrict the PR to actual fixes we can move forward, otherwise it'll be best to close this.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (ItsAdityaKSingh) [reply README.md]: That was a space formatting mistake by my IDE. I have reversed it back. I am really sorry for the troubles and appreciate your guidance.

**Review by stuartmorgan — APPROVED** [review]
LGTM

@gaaclarke for secondary review

**Review by gaaclarke — APPROVED** [review]
